### DISCUSSION TITLE
Using Llama 3.2 or any open source model

### DISCUSSION THREAD

**Discussion by TafadzwaD** [TafadzwaD/artistaa-whatsapp-chatbot#3] [Ideas]
https://github.com/TafadzwaD/artistaa-whatsapp-chatbot/issues/1

**Comment by grtsnx** [comment]
cool.. by the way you may want to check this out;

- https://youtu.be/qY1W1iaF0yA

**Reply by TafadzwaD** [reply]
@thegrtnx I made a video a while back that covers using Ollama for local LLMs, which you can check out here: https://youtu.be/uAgRXQGCCyY. While Ollama is currently suitable for development purposes, it's not yet ready for production use (see more details here: https://github.com/ollama/ollama/issues/3341)

**Reply by TafadzwaD** [reply]
@thegrtnx The video is scheduled for publishing on December 10th! 🎉 Thanks so much for requesting it
